src/porofluid_pressure_based_elast_scatra/4C_porofluid_pressure_based_elast_scatra_artery_coupling_nodetopoint.hpp

```diff
@@ -77,27 +78,27 @@ namespace PoroPressureBased
      * (Hagen-Poiseuille)
      * \note not possible for node-to-point formulation since variable diameter not yet possible
      */
-    void evaluate_additional_linearizationof_integrated_diam() override
+    void evaluate_additional_linearization_of_integrated_diameter() override
     {
       FOUR_C_THROW(
-          "Function 'evaluate_additional_linearizationof_integrated_diam()' not possible for "
+          "Function 'evaluate_additional_linearization_of_integrated_diameter()' not possible for "
           "node-to-point coupling");
     };
 
     /*!
-     * @brief get the segment lengths of element 'artelegid'
+     * @brief get the segment lengths of element 'artery_ele_gid'
```

Comment:
```suggestion
     * @brief get the segment lengths of element @p artery_ele_gid
```

This formats the following word in a monospaced font to look like code.